core/src/test/java/com/github/phantazmnetwork/core/gui/BasicSlotDistributorTest.java

```diff
@@ -0,0 +1,148 @@
+package com.github.phantazmnetwork.core.gui;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class BasicSlotDistributorTest {
+    @Test
+    void test3x3_1Item_padding0() {
```

Comment:
why 1 before item but 0 after padding?

Reply:
added the padding after the XItem part, don't want to go back and change all 69 tests xd

Reply:
come on this is an easy ctrl+shift+r

Reply:
no, requires regex and manually making sure the right number is moved over, or multiple replaces

Reply:
make vince do it then